dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -2128,28 +2137,63 @@ public Node visitSwitch(SwitchTree tree, Void p) {
    */
   private class SwitchBuilder {
     /** The switch tree. */
```

Comment:
Javadoc above says this nested class is only for switch statements; I think that should be updated, and this Javadoc should say "It might be a switch statement or a switch expression."